### PR TITLE
update trial 2 metadata

### PR DESCRIPTION
**Pull request by wholmgren**

### CODE REVIEW

**Comment by alorenzo175** [comment]
I guess this is good with me as is since the trial page/post should go into other details.

File: documentation/dashboard/trials.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+Forecast vendors may upload data for intervals beyond what is required at any
+given time. The data may be overwritten until the closing time of the forecast
+run. Consider a 1 hour lead time, 1 hour interval forecast with
+hour-beginning interval labels. In this example, a forecast vendor need only
+upload a single value each at each hour of the trial.
+For instance, by ``2020-01-01T00:00Z`` a vendor may upload the forecast
+
+```
+# valid upload at any time until 2020-01-01T00:00Z
+time,value
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:142]:
```suggestion
timestamp,value
```

File: documentation/dashboard/trials.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+2020-01-01T01:00Z,10
+```
+
+The vendor could choose to upload additional forecast values such as
+
+```
+# valid upload at any time until 2020-01-01T00:00Z
+time,value
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:150]:
```suggestion
timestamp,value
```

File: documentation/dashboard/trials.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+2020-01-01T03:00Z,30
+```
+
+Now imagine we advance in time to the next forecast interval.
+Between ``2020-01-01T00:00Z`` and ``2020-01-01T01:00Z`` the vendor may change
+the value of all forecasts after ``2020-01-01T01:00Z``. A valid upload could be
+
+```
+# valid upload at any time until 2020-01-01T01:00Z
+time,value
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:162]:
```suggestion
timestamp,value
```

File: documentation/dashboard/trials.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+2020-01-01T04:00Z,45
+```
+
+The API will reject an upload that contains a time that should have already
+been issued. For example, during this interval, the upload would be rejected
+if it contains any times before ``2020-01-01T02:00Z``.
+
+```
+# invalid upload after 2020-01-01T00:00Z
+time,value
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:174]:
```suggestion
timestamp,value
```

File: documentation/dashboard/trials.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+2020-01-01T02:00Z,25
+2020-01-01T03:00Z,35
+2020-01-01T04:00Z,45
+```
+
+Assuming no additional uploads, the time series used for evaluation would be
+
+```
+# forecast evaluation time series
+time,value
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:185]:
```suggestion
timestamp,value
```

File: documentation/dashboard/trials.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+
+Anonymous accounts
+------------------
+{: .anchor}
+
+The Solar Forecast Arbiter will create a new anonymous user account for each
+vendor for each trial. The mapping between the trial participants and the
+anonymized names is created automatically so that the administrators do not know
+the true identities.
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:98]:
```suggestion
the true identities. An automated email will be sent to each trial participant with 
the anonymous username, a password reset link, and a securely signed text file.
This text file can be used to verify that a trial participant was assigned a  
particular anonymous user account, and it should be kept in a safe
location.
```

File: documentation/dashboard/trials.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+* All metadata needed instantiate the relevant Site, Observation, and Forecast
+  objects in the Solar Forecast Arbiter or links to existing metadata. This
+  fully defines the forecast specifications.
+* All metadata needed to instantiate reference forecast objects or links to
+  existing reference forecasts.
+* All metrics to be computed.
+* All categories over which metrics will be calculated.
+* Data quality flags that will be excluded from analysis.
+* Rule for missing forecasts (exclude, fill with 0, fill with last valid value).
+* Rule for visibility of forecast time series with other anonymous vendors.
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:85]:
```suggestion
* Rules for visibility of forecast time series with other anonymous vendors, including 
  during the trial and  in the final report.
```

File: documentation/dashboard/trials.md [hunk 8]
```diff
@@ -0,0 +1,312 @@
...
+```
+
+The script should be run periodically to generate new forecasts, either using
+cron jobs or a cron Python framework like
+[schedule](https://schedule.readthedocs.io/en/stable/). Further
+documentation for the solarforecastarbiter-core Python package can be
+found at
+[https://solarforecastarbiter-core.readthedocs.io/en/latest/](https://solarforecastarbiter-core.readthedocs.io/en/latest/).
+
+<script src="https://gist.github.com/alorenzo175/93ce302e23821bc6f6a78124f135aebc.js"></script>
```
Comment (alorenzo175) [inline documentation/dashboard/trials.md:312]: Should we just add the file to this repo?

Comment (alorenzo175) [reply documentation/dashboard/trials.md:312]:
```suggestion
<script src="https://gist.github.com/alorenzo175/d25293c4e47bec307ede7cd70022582c.js"></script>
```

Comment (wholmgren) [reply documentation/dashboard/trials.md:312]: sure. do you want to do that in a separate PR and then I'll link to it here?

Comment (alorenzo175) [reply documentation/dashboard/trials.md:312]: #158 hmm, I guess we can't embed it nicely like you can with gists

Comment (wholmgren) [reply documentation/dashboard/trials.md]: I have a slight preference for leaving it as the gist for now.

**Comment by wholmgren** [comment]
@alorenzo175 good to merge?

**Pull request merged**